scripts/clean_all.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+# -e: exits if a command fails
+# -u: errors if an variable is referenced before being set
+# -o pipefail: causes a pipeline to produce a failure return code if any command errors
+
+readonly RULES_NODEJS_DIR=$(cd $(dirname "$0")/..; pwd)
+cd ${RULES_NODEJS_DIR}
+
+echo_and_run() { echo "+ $@" ; "$@" ; }
```

Comment:
why not just `set -x` to echo commands before they're run?

Reply:
I wanted to be selective on which ones are echoed and bracketing with `set -x` and `set +x` seemed noisier.